pyproject.toml

```diff
@@ -0,0 +1,97 @@
+[build-system]
+requires = [
+    "black",
+    "isort",
+    "pyproject-flake8",
```

Comment:
Shouldn't this be replaced by ruff?

Reply:
Yes, my oversight. Was pulling a late night.